### ISSUE TITLE
Device configuration, dataset and method.

### ISSUE THREAD

**Issue by MPCheng-ZW** [YoushaaMurhij/OFMPNet#2]
Thanks for your excellent work.

I have three issues to consult with you.

1. What are the specific configurations of your computer? For example, the GPU model, disk memory, etc. By the way, my device has an NVIDIA 4090 GPU and 2TB disk memory. Can I run your code successfully?
2. Which version of the Waymo open motion dataset do you use? For instance, v2.0.1, v2.2.0, etc. Additionally, how much memory does the dataset occupy in total?
3. What’s the difference between OFMPNet and Strajnet? Perhaps the OFMPNet-Swin/LSTM is similar to Strajnet?

Thanks again!

**Comment by YoushaaMurhij** [comment]
1- My configurations are Tesla V100 gpus, 20TB disk and 128GB of RAM. I think 2TB is not enough due to the large size of WOD.
2- I used `waymo_open_dataset_motion_v_1_1_0` as illustrated in data prepartion section. The original dataset occupies > 1TB. And you need more space for preprocessing steps.
3- They share the same encoder structure. There are some changes in the decoder and loss functions. More details can be found in the paper.

**Comment by MPCheng-ZW** [comment]
Thanks for your prompt reply.

Could you please provide the information on the total memory occupied by both the original dataset and preprocessing data? 

Without considering disk memory limitations, can I run the code on the device equipped with an NVIDIA RTX 4090 GPU?

Thank you very much!

**Comment by MPCheng-ZW** [comment]
> Thanks for your prompt reply.
> 
> Could you please provide the information on the total memory occupied by both the original dataset and preprocessing data?
> 
> Without considering disk memory limitations, can I run the code on the device equipped with an NVIDIA RTX 4090 GPU?
> 
> Thank you very much!

**Comment by YoushaaMurhij** [comment]
I do not remember the exact number but I am bretty sure it was more that 2TB. A quick way to know is to downlaod one senario file and run prepocessig on it to estimate the required space for the whole dataset. 
I do not see any problem running this code on 4090. You maybe need to adjust the batch size.

**Comment by MPCheng-ZW** [comment]
My sincere thanks to you!

I see that you train the model on 6 Tesla V100. Is it feasible if I use single RTX 4090 GPU to train the model?

If it is feasible, I plan to set the batch size to 1. Will this effect the performance of the model?

**Comment by YoushaaMurhij** [comment]
I am not quite sure about using only one RTX 4090 to train the model. You should then consider a batch size of 2 and expect a drop in the metrics. I recommend using a bigger batch size.

**Comment by MPCheng-ZW** [comment]
Okay, I got it.

Thanks for your response.

**Comment by MPCheng-ZW** [comment]
Sorry to bother you again.

Can I use a small or mini version of WOD to debug the code?  For example, can I only use the tranning_0000.tar, validation_0000.tar, and testing_0000.tar files? Any advice would be greatly appreciated.

Thank you very much!

**Comment by YoushaaMurhij** [comment]
Sure. You need to prepare them to have the structure in the Data preprocessing section in README.

**Comment by MPCheng-ZW** [comment]
I got it, thanks!
